### DISCUSSION TITLE
Drag and drop windows with mouse?

### DISCUSSION THREAD

**Discussion by Okanda** [YaLTeR/niri#334] [Q&A]
Hello. 

I'm trying Niri and it's really cool. 
I know it is done to be used with keyboard but I wonder if it's possible to drag en drop windows to arrange them with Meta+mouse (or other key)? 
If not will it be implemented in the futur? 

Thank you. 

LCG

edit : add "to arrange them"

**Comment by YaLTeR** [comment]
It will be implemented in the future (with the floating layer at least: #122).

**Comment by Okanda** [comment]
Great! 
Didn't make the link between the two subjects. 

Thank you.

**Reply by YaLTeR** [reply]
I now implemented interactive mouse resizing; it's not movement but it's the other half of that equation, so I figured you might be interested :)

**Comment by Okanda** [comment]
Just saw that this morning. Great. Thank you! 
Hope Arch package will be upgraded soon to test this.
